packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -3,26 +3,24 @@
 // Omitting rgb "fire" style, so no `chalk` dependency
 // Omitting: NaN handling, special consideration for constant data.
 
-let create = (
-  numbers: array<float>,
-  ~minimum=?,
-  ~maximum=?,
-  ()
-) => {
-  let ticks = [`▁`, `▂`, `▃`, `▄`, `▅`, `▆`, `▇`, `█`]
+let ticks = [`▁`, `▂`, `▃`, `▄`, `▅`, `▆`, `▇`, `█`]
 
-  let minimum = E.O.default(Js.Math.minMany_float(numbers), minimum)
-  let maximum = E.O.default(Js.Math.maxMany_float(numbers), maximum)
+let _ticksLength = E.A.length(ticks)
 
-  let toHeight = (number: float) => {
-    let tickIndex = Js.Math.ceil_int((number /. maximum) *. (ticks -> Belt.Array.length -> Belt.Int.toFloat)) - 1
-    let tickIndex = if maximum == 0.0 || tickIndex < 0 {
-        0
-      } else {
-        tickIndex
-      }
-    ticks[tickIndex]
-  }
+let _heightToTickIndex = (maximum: float, v: float) => {
+  let v = Js.Math.ceil_int(v /. maximum *. Belt.Int.toFloat(_ticksLength)) - 1
+  min(v, 0)
```

Comment:
am I off my rocker or should this be `max`?

Reply:
Weird, I'm pretty sure you're right. Now I'm paranoid about the tests

Reply:
yeah in the case when `v == 0` the `ceil` will be zero, and the new `v` (or as I'm suggesting `w`) will be `-1`, then `min(-1, 0) == -1`, which will get interpreted as an index, raising an error.